src/main/java/com/corndel/cashregister/Level4.java

```diff
@@ -10,6 +10,30 @@ public class Level4 {
    */
   public static boolean canMakeAmount(int target, List<Item> drawer) {
     // TODO
-    return false;
+    boolean targetAchieved = false;
+
+   for(Item currentCash : drawer){
+    int value = currentCash.getValue();
```

Comment:
This is great! good use of variables to avoid un intentionally modifying the items quantity instance variables!